input/input.h

```diff
@@ -11,7 +11,26 @@
 
 #include "defines.h"
 
-
+/**
+ * Waits for user input and returns the input
+ * as an ASCII character
+ */
```

Comment:
Needs an `@return` to say what the return value is

Reply:
Fixed